test/testcafe/framework/page-objects/BasePageObject.js

```diff
@@ -100,30 +101,40 @@ export default class BasePageObject {
     return this.form.getElement(ionMessagesSelector).innerText;
   }
 
+  getCancelLink() {
+    return this.form.getLink(CANCEL_LINK_TEXT);
+  }
```

Comment:
Can we leave a comment here or maybe below explaining that in v3, the `Cancel Link` actually covers multiple links, like "Go Back" and "Sign out"?

Reply:
okay will do but actually its v2 that covers multiple links.  In v3 its only "Back to sign in"

Reply:
Oh ok yeah bad wording on my part, that's what I meant, that in v3 they are all the same wording